pkgs/tools/compression/bsdiff/CVE-2020-14315.patch

```diff
@@ -0,0 +1,384 @@
+Description: patch for CVE-2020-14315
```

Comment:
Where is this file being used :thinking: ?

Reply:
My bad, I forgot to commit the changes in default.nix